src/ray/core_worker/actor_manager.cc

```diff
@@ -163,9 +163,7 @@ void ActorManager::HandleActorStateNotification(const ActorID &actor_id,
                 << ClientID::FromBinary(actor_data.address().raylet_id())
                 << ", num_restarts: " << actor_data.num_restarts();
 
-  if (actor_data.state() == gcs::ActorTableData::PENDING) {
```

Comment:
I think we should check both PENDING and UNRESOLVED here and do nothing here? Otherwise, it will try to connect to an actor that is not created

Reply:
> I think we should check both PENDING and UNRESOLVED here and do nothing here? Otherwise, it will try to connect to an actor that is not created

Yes, my bad, I will fix it soon.